karabo_bridge/simulation.py

```diff
@@ -283,6 +287,14 @@ def start_gen(port, ser='msgpack', version='2.2', detector='AGIPD',
         Number of sources.
     datagen: string, optional
         Generator function used to generate detector data. Default is random.
+    frame: string optional ['online', 'file']
```

Comment:
The name 'frame' doesn't convey this very clearly to me - though maybe I've been thinking about it too long from a data perspective to know what a scientist seeing it for the first time would think.

How about something like `axis_order` or `dim_order` or `data_like`?

Reply:
I also did not like it and hoped for someone to suggest something better, thanks :)